src/languageHandler.tsx

```diff
@@ -53,9 +53,9 @@ interface ITranslatableError extends Error {
  * @param {string} message Message to translate.
  * @returns {Error} The constructed error.
  */
-export function newTranslatableError(message: string) {
+export function newTranslatableError(message: string, variables?: IVariables): ITranslatableError {
```

Comment:
wondering if we should shorten this to match other funcs, maybe `_tError` or similar

Reply:
`_tError` seems reasonable 👍 but I'll leave that taste making to you for another PR